node/lib/openshift-origin-node/model/node.rb

```diff
@@ -336,6 +337,8 @@ def self.node_utilization
 
         # use max_{active_,}gears if set in resource limits, or fall back to old "apps" names
         res['max_active_gears'] = (resource.get('max_active_gears') or resource.get('max_active_apps') or DEFAULT_MAX_ACTIVE_GEARS)
+        res['node_df_buffer'] = resource.get('node_df_buffer', DEFAULT_NODE_DF_BUFFER)
```

Comment:
It would make sense to do a `.to_f` here so that the `facts.yaml` doesn't sometimes contain a string (if the value from the configuration file is used) and sometimes a float (if `DEFAULT_NODE_DF_BUFFER` is used), although it shouldn't really matter since we do a `.to_f` on the broker's side.